src/hooks/screen/useSignedChannelListHook.ts

```diff
@@ -35,7 +35,7 @@ function useSignedChannelListScreenHook(): UseSignedChannelListScreenHook {
       response?.message,
       null,
       true,
-      response?.duration
+      response?.expired_at
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `response?.duration` to `response?.expired_at` seems to indicate a shift in the type of data being handled. Ensure that all parts of your application that use this function are updated to handle an expiration timestamp instead of a duration. If there's any logic that depends on the duration, you'll need to calculate it based on the `expired_at` value.
```diff
-      response?.duration
+      response?.expired_at
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->